contrib/k8s-lbry/run.sh

```diff
@@ -0,0 +1,661 @@
+            WALLET=/home/lbrynet/.local/share/lbry/lbryum/wallets/default_wallet
+            KUBECTL=$(which kubectl)
+            POD=$($KUBECTL -n "$NAMESPACE" get pod -l app=speech -o name | sed s+pod/++)
+            kubectl cp "$POD":$WALLET "$1" -c speech-lbrynet
+            chmod 600 "$1"
+            echo "lbrynet wallet copied to $1"
+        else
+            echo "Required arg: path of backup location for wallet"
+        fi
+    )
+}
+
+lbrycrd-cli() {
+    ## Run lbrycrd-cli client from inside the running pod outputting to your local terminal
+    KUBECTL=$(which kubectl)
+    POD=$($KUBECTL -n "$NAMESPACE" get pod -l app=lbrycrd -o name | sed s+pod/++)
+    if [ ${#POD} -gt 0 ]; then
+        kubectl exec "$POD" -- lbrycrd-cli -rpcuser="$LBRYCRD_RPC_USER" -rpcpassword="$LBRYCRD_RPC_PASSWORD" "$*"
+    else
+        echo "lbrycrd pod not found"
+    fi
+}
+
+lbrynet() {
+    ## Run lbrynet client from inside the running pod outputting to your local terminal
+    KUBECTL=$(which kubectl)
+    POD=$($KUBECTL -n "$NAMESPACE" get pod -l app=lbrynet -o name | sed s+pod/++)
+    if [ ${#POD} -gt 0 ]; then
+        kubectl exec "$POD" -- lbrynet "$*"
+    else
+        echo "lbrynet pod not found"
+    fi
+}
+
+speech-lbrynet() {
+    ## Run lbrynet client from inside the running pod outputting to your local terminal
+    KUBECTL=$(which kubectl)
+    POD=$($KUBECTL -n "$NAMESPACE" get pod -l app=speech -o name | sed s+pod/++)
+    if [ ${#POD} -gt 0 ]; then
+        kubectl exec "$POD" -c speech-lbrynet -- lbrynet "$*"
+    else
+        echo "lbrynet pod not found"
+    fi
+}
+
+SUBCOMMANDS_NO_ARGS=(setup install install-nginx-ingress install-cert-manager upgrade
+                     upgrade-nginx-ingress upgrade-cert-manager chainquery-mysql-client
+                     speech-mysql-client chainquery-override-snapshot lbrycrd-override-snapshot
+                     setup-alias)
+
+SUBCOMMANDS_PASS_ARGS=(helm kubectl shell shell-pvc restart package logs lbrynet-copy-wallet lbrynet speech-lbrynet-copy-wallet speech-lbrynet lbrycrd-cli completion)
+
+completion() {
+    if [ "$#" -eq 1 ] && [ "$1" == "bash" ]; then
+        cat <<EOF
+__delegate_k8s_lbry() {
+    alias kubectl="kubectl --namespace=$NAMESPACE"
+    local cur subs
+    cur="\${COMP_WORDS[COMP_CWORD]}" # partial word, if any
+    subs="${SUBCOMMANDS_NO_ARGS[*]} ${SUBCOMMANDS_PASS_ARGS[*]}"
+    if [[ \${COMP_CWORD} -gt 1 ]]; then
+        # complete subcommands
+        _command \${@: -1}
+    else
+        # complete with the list of subcommands
+        COMPREPLY=( \$(compgen -W "\${subs}" -- \${cur}) )
+    fi
+}
+if [[ $(type -t compopt) = "builtin" ]]; then
+    complete -o default -F __delegate_k8s_lbry $RUN_ALIAS
+else
+    complete -o default -o nospace -F __delegate_k8s_lbry $RUN_ALIAS
+fi
+EOF
+    else
+        echo "## I only know how to do completion for the bash shell."
+        echo "## Try '$0 completion bash' instead."
+    fi
+
+}
+
+setup-alias() {
```

Comment:
This should verify that the user's shell is BASH if you're planning on adding this.

Reply:
Now it does. thank you.